src/app/retrospective/sprint-detail/sprint-detail.component.ts

```diff
@@ -62,6 +63,12 @@ export class SprintDetailComponent implements OnInit, OnDestroy  {
 
     ngOnInit() {
         const params = this.activatedRoute.snapshot.params;
+        const autoRefreshState = JSON.parse(localStorage.getItem(AUTO_REFRESH_TOKEN_KEY));
```

Comment:
Let's discuss if we should use sessionStorage instead of localStorage (Refer: https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage)

Reply:
As I have already mentioned in my [JIRA ticket](https://ireflect.atlassian.net/browse/IR-227), we use local storage instead of session storage to persist user's Autorefresh state even after page session ends.

The Page you linked above clearly mentions this: `Opening a page in a new tab or window will cause a new session to be initiated`